src/cli/commands/manager/list_checkpoints.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! List checkpoints cli command
+
+use async_trait::async_trait;
+use clap::Args;
+use fvm_shared::clock::ChainEpoch;
+use ipc_gateway::Checkpoint;
+use ipc_sdk::subnet_id::SubnetID;
+use std::collections::HashMap;
+use std::fmt::Debug;
+
+use crate::cli::commands::get_ipc_agent_url;
+use crate::cli::{CommandLineHandler, GlobalArguments};
+use crate::config::json_rpc_methods;
+use crate::jsonrpc::{JsonRpcClient, JsonRpcClientImpl};
+use crate::server::list_checkpoints::ListCheckpointsParams;
+
+/// The command to list checkpoints committed in a subnet actor.
+pub(crate) struct ListCheckpoints;
+
+#[async_trait]
+impl CommandLineHandler for ListCheckpoints {
+    type Arguments = ListCheckpointsArgs;
+
+    async fn handle(global: &GlobalArguments, arguments: &Self::Arguments) -> anyhow::Result<()> {
+        log::debug!("list checkpoints with args: {:?}", arguments);
+
+        let url = get_ipc_agent_url(&arguments.ipc_agent_url, global)?;
+        let json_rpc_client = JsonRpcClientImpl::new(url, None);
+
+        let params = ListCheckpointsParams {
+            subnet_id: arguments.subnet_id.clone(),
+            from_epoch: arguments.from_epoch,
+            to_epoch: arguments.to_epoch,
+        };
+
+        let checkpoints = json_rpc_client
+            .request::<HashMap<SubnetID, Checkpoint>>(
```

Comment:
Typo? The key of the hashmap should be the epoch, right? 
```suggestion
            .request::<HashMap<Epoch, Checkpoint>>(
```

Reply:
Fixed. This actually should be Vec<Checkpoint>